commands/publish/index.js

```diff
@@ -140,13 +140,21 @@ class PublishCommand extends Command {
 
     tasks.push(() => this.updateUpdatedPackages());
 
+    if (!this.gitEnabled) {
+      this.logger.info("Skipping git...");
```

Comment:
Oh yeah, I forgot, `npmlog` takes a `prefix` argument in addition to the `message`. The loggingOutput helper skips the prefix, focusing on the message. Sorry for the hassle.

I'd prefer these logs to be added to the existing conditional wrappers instead of separate. Reads a bit cleaner.

```diff
diff --git a/commands/publish/index.js b/commands/publish/index.js
index 5f83e350..59a65d04 100644
--- a/commands/publish/index.js
+++ b/commands/publish/index.js
@@ -154,12 +154,16 @@ class PublishCommand extends Command {
 
     if (this.gitEnabled) {
       tasks.push(() => this.commitAndTagUpdates());
+    } else {
+      this.logger.info("execute", "Skipping git commit/push");
     }
 
     tasks.push(() => this.resolveLocalDependencyLinks());
     tasks.push(() => this.annotateGitHead());
 
-    if (!this.options.skipNpm) {
+    if (this.options.skipNpm) {
+      this.logger.info("execute", "Skipping publish to registry");
+    } else {
       tasks.push(() => this.publishPackagesToNpm());
     }
 ```

The changes to the log message reflect the _actual_ intention of the flags (`--skip-git` doesn't skip _all_ `git` executions, for example).